neural_speed/core/ne_layers.c

```diff
@@ -707,6 +709,9 @@ static inline int ne_up(int n, int m) {
   return (n + m - 1) & ~(m - 1);
 }
 
+// static inline  void ne_vec_tanh_f32 (const int n, float * y, const float * x) { for (int i = 0; i < n; ++i) y[i] =
```

Comment:
why disable this function?

Reply:
This function is redefined, I will remove it.